src/cpp/src/image_generation/models/autoencoder_kl.cpp

```diff
@@ -255,6 +255,7 @@ ov::Tensor AutoencoderKL::decode(ov::Tensor latent) {
 }
 
 ov::Tensor AutoencoderKL::encode(ov::Tensor image, std::shared_ptr<Generator> generator) {
+    OPENVINO_ASSERT(m_encoder_request || m_encoder_model, "AutoencoderKL is created without 'VAE encoder' capability. Please, pass extra argument to constructor to create 'VAE encoder'");
```

Comment:
Do we need both of these asserts there? Isn't the second one not enough?

Reply:
second one will print `VAE encoder model must be compiled first. Cannot infer non-compiled model`, which is less informative when user just missed VAE encoder model in ctor.